src/core/bias-core.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'lodash';
+import BPromise from 'bluebird';
+const {knex} = require('../util/database').connect();
+
+
+const calculateBias = (opts) => {
```

Comment:
To follow convention the param should not be in parenthesis (dunno why there isn't but there should be an eslint config for this)